scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -184,11 +185,17 @@ def write_tie_out_vcf(vds, vcf_output_path):
     parser.add_argument('--ancestry_input_path', type=str, help='Input ancestry file path', required=True)
     parser.add_argument('--vds_input_path', type=str, help='Input VDS path', default="@VDS_INPUT_PATH@")
     parser.add_argument('--sites_only_output_path', type=str, help='Output sites-only VCF path',
-                        default="@SITES_ONLY_VCF_OUTPUT_PATH@")
+                        default="@SITES_ONLY_VCF_OUTPUT_PATH@"),
+    parser.add_argument('--temp_path', type=str, help='Path to temporary directory', default="@TEMP_DIR@",
+                        required=True)
 
     args = parser.parse_args()
 
-    vds = hl.vds.read_vds(args.vds_path)
-    local_ancestry_file = create_vat_inputs.download_ancestry_file(args.ancestry_file)
+    temp_path = args.temp_path if not args.temp_path.endswith('/') else args.temp_path[:-1]
+    time_stamp = datetime.today().strftime('%Y-%m-%d_%H:%M:%S')
```

Comment:
it is [strongly recommended](https://cloud.google.com/storage/docs/objects#naming) not to use colons in GCS object names (yeah for Windows reasons, but still)

Reply:
Thanks. It's a darn shame though.